sites/all/modules/hr/hr_documents/hr_documents.module

```diff
@@ -95,3 +95,20 @@ function hr_documents_ctools_plugin_directory($module, $plugin) {
   }
 }
 
+/**
+ * Implements hook_field_widget_WIDGET_TYPE_form_alter().
+ */
+function hr_documents_field_widget_form_alter(&$element, &$form_state, $context) {
```

Comment:
To fix the dropdown on hr-document edit, see https://humanitarian.atlassian.net/browse/HRINFO-389